### PR TITLE
Add release information about Kubeflow 1.10

### PR DESCRIPTION
**Pull request by rimolive**
Solves #[762](https://github.com/kubeflow/community/issues/762)

### CODE REVIEW

**Comment by rimolive** [comment]
/hold

Waiting on missing information

**Comment by rimolive** [comment]
/unhold

@kubeflow/kubeflow-steering-committee May I ask for review/approval?

File: releases/release-1.10/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Kubeflow 1.10
+
+### Links
+
+- [Release Team](release-team.md)
+- [Calendar](https://www.kubeflow.org/docs/about/community/#kubeflow-community-calendars)
```
Comment (andreyvelich) [inline releases/release-1.10/README.md:6]: This link, should be fixed with: https://www.kubeflow.org/docs/about/community/#kubeflow-community-meetings

File: releases/release-1.10/release-team.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Kubeflow 1.10 Release Team
+
+| **Role** | **Name** (**GitHub / Slack ID**) |
+|----------|----------------------------------|
+| Release Manager | Ricardo Martinelli (Github: [@rimolive](https://github.com/rimolive) / Slack: `Ricardo Martinelli de Oliveira`)|
+| Release Manager  Shadows | Matteo Morttari (GitHub: [@tarilabs](https://github.com/tarilabs) / Slack: `Matteo Morttari`) <br> Julius von Kohout (Github: [@juliusvonkohout](https://github.com/juliusvonkohout) / Slack: `Julius von Kohout`) <br> Valentina Rodriguez Sosa (GitHub: [@varodrig](https://github.com/varodrig) / Slack: `Valentina Rodriguez Sosa`) <br> Vraj Bhatt (GitHub: [@dpoulopoulos](https://github.com/dpoulopoulos) / Slack: `Vraj Bhatt`) |
+| Docs Lead | Diego Lovison (Github: [@diegolovison](https://github.com/diegolovison)) / Slack: `Diego Lovison` |
+| Test Lead | Dagvanorov Lkhagvajav (GitHub: [@tombuuz](https://github.com/tombuuz) / Slack: `Dagvanorov`) |
+| Product Manager | Dimitris Poulopoulos (GitHub: []() / Slack: `Dimitris Poulopoulos`) |
```
Comment (andreyvelich) [inline releases/release-1.10/release-team.md:9]: Do we have GitHub handle for Dimitris ?

File: releases/release-1.10/release-team.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+| AutoML and Training WG Liaison(s) | Sailesh Duddupudi (GitHub: [@saileshd1402](https://github.com/saileshd1402) / Slack `Sailesh Duddupudi`) |
+| Notebooks WG Liaison(s) | Manos Vlassis (GitHub: [@mvlassis](https://github.com/mvlassis) / Slack: `Manos Vlassis`) |
+| Platform WG Liaison(s) | Tarek Abouzeid (GitHub: [@tarekabouzeid](https://github.com/tarekabouzeid) / Slack: `Tarek Abouzeid`) |
+| Pipelines WG Liaison(s) | Helber Belmiro (Github: [@hbelmiro](https://github.com/hbelmiro)) / Slack: `Helber Belmiro` |
+| Data WG Liaison(s) | Milos Grubjesic (GitHub: [@milosjava](https://github.com/milosjava) / Slack: `Milos Grubjesic` ) |
+| KServe Liaison(s) | Vraj Bhatt (GitHub: [@vrajjbhatt](github.com/vrajjbhatt) / Slack: `Vraj Bhatt` |
+
+- The definition of the release team roles and responsibilities can be found at [release handbook](../handbook.md)
+- The timeline for the release can be found at [pending](README.md)
```
Comment (andreyvelich) [inline releases/release-1.10/release-team.md:18]: Should this be renamed ?
```suggestion
- The timeline for the release can be found [here](README.md).
```

**Review by andreyvelich — COMMENTED** [review]
Thanks @rimolive!

**Comment by rimolive** [comment]
@andreyvelich Thank you! Those are nice catches and I didn't noticed them. I just added the proposed changes.

**Pull request merged**